components/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,76 @@
+import { PropTypes, Component } from 'react'
+
+require('./MenuBar.scss')
+
+class MenuBar extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {mobile: false}
+  }
+
+  componentWillMount() {
+    this.handleResize(this)
+    const handleResize = this.handleResize
+    const thisRef = this
```

Comment:
We can use `.bind()` or ES6 arrow functions to preserve the correct `this` value.